packages/dds/tree/src/feature-libraries/forest-proxy/forestProxy.ts

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FieldKey, EmptyKey } from "../../tree";
+import { IEditableForest, TreeNavigationResult, ITreeSubscriptionCursor } from "../../forest";
+
+export const proxySymbol = Symbol("forest-proxy");
+
+class TargetForest {
+	public readonly cursor: ITreeSubscriptionCursor;
+	private readonly _children: Map<string, any>;
```

Comment:
we should avoid the `any` type and instead use more specific types (especially outside of the internals of functions)

If there really is a reason to use the `any`, the field really needs to document what is allowed to go there.